tests/ui/docker-compose.functional-tests.yml

```diff
@@ -5,4 +5,4 @@ version: "2.3"
 
 services:
   addons-frontend:
-    build: ../.
+    image: addons-frontend:latest
```

Comment:
Since this *looks* like a docker hub URL, could you add a comment saying that it will actually trigger a local build?